### DISCUSSION TITLE
Unable To Uninstall Blackhole After Migration To macOS Sequoia

### DISCUSSION THREAD

**Discussion by spiffy-unwind-oops** [ExistentialAudio/BlackHole#811] [Q&A]
Hi,

I just migrated to macOS Sequoia and I cannot uninstall Blackhole (or even find the drivers for that matter). I can see Blackhole in the Audio MIDI Setup app but when I run the uninstaller it fails and when I follow the instructions for manual uninstallation there is no blackhole2ch.driver in the HAL directory. I even did a full-disk search for blackhole* and all I find are the original installation files that I downloaded. I appreciate any help you can provide.

Thanks,
Rob
<img width="903" alt="audio_midi_setup_screenshot" src="https://github.com/user-attachments/assets/55eca28c-f26f-4d27-9347-dd8825fe495b">

**Comment by devinroth** [comment]
Do you have any Rogue Ameoba’s stuff installed? If so, that’s the usual culprit. Remove that and BlackHole will disappear.

Devin

**Reply by spiffy-unwind-oops** [reply]
Hi, Devin,

I have no Rogue Amoeba software installed.

Best,
Rob

**Comment by devinroth** [comment]
Assuming you already restarted your computer, send me a screenshot of the HAL folder.

Devin

**Comment by spiffy-unwind-oops** [comment]
Hi, Devin,

Here you go.

Best,
Rob
<img width="870" alt="hal_folder_screenshot" src="https://github.com/user-attachments/assets/1ddfae61-c736-4ccc-a355-2339d176f1f9">

**Comment by devinroth** [comment]
I don’t know. Try restarting again.

It’s not suppose to exist if it’s not in the HAL folder.

Maybe try rebooting CoreAudio.

Does it show up in audio apps? If no maybe Audio MIDI setup isn’t updating properly.

Devin

**Comment by spiffy-unwind-oops** [comment]
Hi, Devin,

Ha, that did it! So, a second reboot and a reboot of CoreAudio. Thanks so much for your help and a very useful driver. I think we have established that macOS Sequoia is still very buggy! ;-))

Take care and have a good one,
Rob

**Comment by devinroth** [comment]
You’re welcome.

That was odd.

Devin
